ipns/errors.go

```diff
@@ -4,41 +4,45 @@ import (
 	"errors"
 )
 
-// ErrExpiredRecord should be returned when an ipns record is
-// invalid due to being too old
-var ErrExpiredRecord = errors.New("expired record")
+// MaxRecordSize is the IPNS Record [size limit].
+//
+// [size limit]: https://specs.ipfs.tech/ipns/ipns-record/#record-size-limit
+const MaxRecordSize int = 10 << (10 * 1)
 
-// ErrUnrecognizedValidity is returned when an IpnsRecord has an
-// unknown validity type.
-var ErrUnrecognizedValidity = errors.New("unrecognized validity type")
+// ErrExpiredRecord is returned when an IPNS [Record] is invalid due to being expired.
+var ErrExpiredRecord = errors.New("the IPNS record is expired")
```

Comment:
nit: (here and all below) – we return multierror + most likely will wrap them upstream, perhaps we can drop "the IPNS" from all of them? 
```suggestion
var ErrExpiredRecord = errors.New("record is expired")
```